clusterman/batch/autoscaler.py

```diff
@@ -97,7 +97,12 @@ def configure_initial(self):
         self.logger = logger
 
         self.apps = [self.options.pool]  # TODO (CLUSTERMAN-126) someday these should not be the same thing
+        self.initialize_autoscaler()
 
+        # We don't want to watch anything here because the autoscaler bootstrap script takes care of that for us
+        self.config.watchers.clear()
```

Comment:
Would this work? I recall that in `yelp_batch`, the watchers property does not deep copy, which means that you'd be clearing a copy here.

Reply:
Yeah, you're right. I think this piece of code was already here initially. I can add some functionality in yelp_batch to support this. However, I don't think it's necessary to clear these watchers at configure_initial when watchers are only loaded in during autoscaler bootstrap anyways.

Reply:
These are necessary to clear, otherwise you have both the autoscaler batch and the bootstrap batch trying to watch these files.  We just want the bootstrap batch to watch.